utils/conda_upload.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# file: conda_upload.sh
+# description: builds and uploads cyclus projects (cyclus, cycamore, and cymetric) to binstar
+# for a release
+
+set -e
+
+usage() {
+
+cat <<EOF
+
+Script to upload compiled binaries of Cyclus, Cycamore, and Cymetric to binstar as part of the release process.
+
+usage: $0 <version>
+
+        version : the version string (e.g., X.Y.Z) to with which this package will be labeled.
+
+The following environment variables must be set for this script to function:
+
+     CYCLUS_DIR : Environment variable CYCLUS_DIR must be set to the cyclus repository directory.
+   CYCAMORE_DIR : Environment variable CYCAMORE_DIR must be set to the cycamore repository directory.
+   CYMETRIC_DIR : Environment variable CYMETRIC_DIR must be set to the cymetric repository directory.
+
+EOF
+
+}
+
+die() {
+    echo -n >&2 "ERROR: "
```

Comment:
done and done

On Fri, May 8, 2015 at 6:17 PM, Anthony Scopatz <notifications@github.com>
wrote:

> In utils/conda_upload.sh
> <https://github.com/cyclus/ciclus/pull/63#discussion_r29984400>:
>
> > +usage: $0 <version>
> > +
> > +        version : the version string (e.g., X.Y.Z) to with which this package will be labeled.
> > +
> > +The following environment variables must be set for this script to function:
> > +
> > +     CYCLUS_DIR : Environment variable CYCLUS_DIR must be set to the cyclus repository directory.
> > +   CYCAMORE_DIR : Environment variable CYCAMORE_DIR must be set to the cycamore repository directory.
> > +   CYMETRIC_DIR : Environment variable CYMETRIC_DIR must be set to the cymetric repository directory.
> > +
> > +EOF
> > +
> > +}
> > +
> > +die() {
> > +    echo -n >&2 "ERROR: "
>
> Inconsistent whitespace.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/ciclus/pull/63/files#r29984400>.
>



-- 
Matthew Gidden, Ph.D.
Postdoctoral Associate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192